### PR TITLE
[chore](cloud) Add show_meta_ranges API

### PR DESCRIPTION
**Pull request by w41ter**
This interface can roughly estimate the storage space occupied by different types of keys by analyzing the FDB partition key.

The usage:

```
curl localhost:5000/MetaService/http/show_meta_ranges?token=greedisgood9999
```

An example:

```
$ curl localhost:5000/MetaService/http/show_meta_ranges?token=greedisgood9999
total partitions: 5
recycle|default_instance_id|txn: 1
meta|default_instance_id|tablet_index: 1
meta|default_instance_id|rowset_tmp: 1
```

### CODE REVIEW

**Comment by w41ter** [comment]
run buildall

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by gavinchou — CHANGES_REQUESTED** [review]

File: cloud/src/meta-service/meta_service_http.cpp [hunk 1]
```diff
@@ -350,6 +355,70 @@ static HttpResponse process_get_value(MetaServiceImpl* service, brpc::Controller
...
+    }
+
+    std::vector<std::string> partition_boundaries;
+    TxnErrorCode code = txn_kv->get_partition_boundaries(&partition_boundaries);
+    if (code != TxnErrorCode::TXN_OK) {
+        auto msg = fmt::format("failed to get boundaries, code={}", code);
+        return http_json_reply(MetaServiceCode::UNDEFINED_ERR, msg);
+    }
+
+    std::unordered_map<std::string, size_t> partition_count;
```
[the remark this answers is not in the document]
Comment (w41ter) [reply cloud/src/meta-service/meta_service_http.cpp:373]: No comment is needed here, partition is a very basic concept.

**Comment by w41ter** [comment]
run buildall

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Pull request merged**